### PR TITLE
IMDSv1 to IMDSv2 on v3 racks

### PR DESCRIPTION
**Pull request by yuriadams**
### What is the feature/fix?

Add field on eks cluster to control if Whether or not the metadata service requires session tokens, also referred to as Instance Metadata Service Version 2 (IMDSv2). Can be optional or required.

Task Link in [Asana](https://app.asana.com/0/1203637156732418/1204066841595405/f)
 
### Does it has a breaking change?

No

### How to use/test it?

There's a new parameter available on rack installation called: imds_http_tokens that can be `optional` or `required`.

### Checklist
- [ ] New coverage tests
- [ ] Unit tests passing
- [ ] E2E tests passing
- [ ] E2E downgrade/update test passing
- [ ] Documentation updated
- [ ] No warnings or errors on Deepsource/Codecov

### CODE REVIEW

**Review by nightfury1204 — DISMISSED** [review]

File: terraform/system/aws/main.tf [hunk 1]
```diff
@@ -55,6 +55,7 @@ module "cluster" {
   key_pair_name            = var.key_pair_name
   kube_proxy_version       = var.kube_proxy_version
   k8s_version              = var.k8s_version
+  imds_http_tokens         = var.imds_http_tokens
```
Comment (nightfury1204) [inline terraform/system/aws/main.tf:58]: did we already has this support?

Comment (nightfury1204) [reply terraform/system/aws/main.tf:58]: sorry i don't see where it is being applied 
i only see these changes: https://github.com/convox/convox/pull/591/files

Comment (yuriadams) [reply terraform/system/aws/main.tf:58]: yes, it's true. Some files are missing. Thank for letting me know! :)

Comment (yuriadams) [reply terraform/system/aws/main.tf:58]: this is a rack parameter, so, the only change is in the tf files, right?

File: terraform/cluster/aws/main.tf [hunk 2]
```diff
@@ -155,6 +155,10 @@ resource "aws_launch_template" "cluster" {
     }
   }
 
+  metadata_options{
```
Comment (nightfury1204) [inline terraform/cluster/aws/main.tf:158]: can you run `terraform fmt`?

**Review by nightfury1204 — APPROVED** [review]

**Pull request merged**